package.json

```diff
@@ -133,7 +133,7 @@
     "eject": "react-scripts eject",
     "start": "yarn lingui:prepare && react-scripts start",
     "test": "react-scripts test",
-    "test:unit": "react-scripts test --watchAll=false --bail --ci --coverage",
+    "test:unit": "react-scripts test --watchAll=false --bail --ci --coverage --detectOpenHandles",
```

Comment:
we removed the `--detectOpenHandles` to make the tests run faster. The idea is we should just be using `--detectOpenHandles` locally.

Reply:
Thanks for the clarification. Reverted.